build/buildRunTimeImages.sh

```diff
@@ -64,6 +64,9 @@ fi
 for generateDockerFile in $generateDockerFiles; do
     echo
     echo "Executing '$generateDockerFile'..."
+	ls -l "$generateDockerFile"
```

Comment:
why twice?

Reply:
ah this needs to be removed … this was for that weird permission issue